projects/cashmere/src/lib/icon-font/hcicons.scss

```diff
@@ -1,3 +1,5 @@
+$hc-icons-font-path: "" !default;
```

Comment:
Found that this path by needed to be ".", because the font is a sibling to this stylesheet within node_modules